code/headers/controller_interface_c.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <joystick_value_s.hpp>
+
+namespace R2D2::manual_control {
```

Comment:
The project namespace is lower case, please change this in the whole repo to `r2d2`